src/reducers/githubAuth.js

```diff
@@ -22,6 +23,13 @@ const githubAuth = (state = initialState, action) => {
         isLoading: true,
         isLoggedIn: false,
       };
+    case AUTHENTICATED:
+      localStorage.setItem("token", JSON.stringify(action.payload.token));
```

Comment:
Ideally, your reducers should only return new state and any side effect should be handled outside of reducers.

Reply:
Okay